src/main/resources/application.properties

```diff
@@ -1 +1,7 @@
-
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=password
```

Comment:
Since it is a test project this is fine but passwords should not be pushed on to the app configs with out encryption